JumpscaleCore/data/bcdb/BCDB.py

```diff
@@ -804,6 +804,10 @@ def _unserialize(self, id, data, return_as_capnp=False, schema=None):
         if return_as_capnp:
             return bdata
         else:
+            if not schema:
```

Comment:
by itself its a good fix but I think the problem is rather how this function is called
shouldn't the schema be there?
but we can leave as is

Reply:
in this case it's called by iterate method, the schema is not available there